### DISCUSSION TITLE
How to hint whether an element has a tippy instance associated with it?

### DISCUSSION THREAD

**Discussion by karlicoss** [atomiks/tippyjs#1100] [Q&A]
TLDR: I'm trying to use Tippy for my browser extension, [promnesia](https://github.com/karlicoss/promnesia#demos)
In essence, I need to attach tooltips to a subset of links on the page, like on [this screenshot](https://camo.githubusercontent.com/18fddd103efebd549a64c9a17d9468ce66ed62bcd01044760e28e88a4cfc9198/68747470733a2f2f6b61726c69636f73732e6769746875622e696f2f70726f6d6e657369612d64656d6f732f73637265656e732f7669736974735f6368696c647669736974735f6c6f6361746f725f706f707570735f626f72696e675f696e746572657374696e672e706e67). Currently I'm doing it with a semitransparent border around the element + eye icon as you can see on the screen.

I managed to show tooltips on hover, but currently there is no way to tell if the link even has a tooltip attached to it (so the user doesn't know what they can hover over in the first place). Is there a native way in Tippy to do that? Went through the docs, but didn't find anything.

I could of course just keep doing that manually, but wonder if there is a native way in Tippy to do that. Feel like a useful feature to have, so I really suspect there must be some way.

Thanks!
